APIs/OpenSearch.md

```diff
@@ -1,6 +1,6 @@
 # OpenSearch
 
-OpenSearch is an open-source software suite for search, analytics, and observability applications. It can be employed as a complete solution or integrated with open-source tools.
+The OpenSearch standard aims to provide an easy way to query a repository containing Earth Observation data and to enable repository syndication.
```

Comment:
I had to look up 'syndication' in a dictionary to be sure of the exact meaning. This got copy pasted from the ogc standard directly again. We'll want to start checking that introductory paragraphs like this one are actually understandable for a broad audience (that's the purpose of an intro).
It's also always good to keep into mind that the majority of our audience are non-native English speakers.